Sources/OneSignal/OneSignalNotification.swift

```diff
@@ -11,6 +11,7 @@ import Vapor
 public struct OneSignalNotification: Codable {
     enum CodingKeys: String, CodingKey {
         case users
+        case iosDeviceTokens
```

Comment:
When I started building out the API, I wasn't even aware this was possible, but I might of just overlooked it. I do know that the preferred way is to use the player_id from the iOS/Android SDK to send notifications. 

Anyways though, looking at this, and knowing how they snake_case everything, I'm leaning towards this having to be 
```suggestion
        case iosDeviceTokens = "ios_device_tokens"
```
 but I'm not sure. I'll take a better look over this in the morning. 👍

Reply:
This is taken care of in:
```
public struct OneSignalPayload: Content {
    enum CodingKeys: String, CodingKey {
        case appId = "app_id"
        case playerIds = "include_player_ids"
        case iosDeviceTokens = "include_ios_tokens"```

Reply:
Awesome!